src/framework/preferences/js/PrefsEditor.js

```diff
@@ -158,7 +184,16 @@ var fluid_2_0 = fluid_2_0 || {};
         var mapped = fluid.transform(that.options.resources, that.transformURL);
 
         return fluid.transform(mapped, function (url) {
-            return {url: url, forceCache: true, options: that.options.resourceOptions};
+            var resourceSpec = {url: url, forceCache: true, options: that.options.resourceOptions};
+            fluid.each(["defaultLocale", "locale"], function (localeName) {
```

Comment:
This can simply be resourceSpec = $.extend(resourceSpec, fluid.filterKeys(that.options, ["defaultLocale", "locale"]));